components/sst_importer/src/sst_importer.rs

```diff
@@ -169,6 +179,26 @@ impl SSTImporter {
             "path" => path_str,
         );
 
+        if is_raw_kv {
+            self.finish_downloading_rawkv::<E>(meta, path, sst_writer, start)
```

Comment:
is there so much difference between the two which requires a separate function? 🤔 The `rewrite_rule` shouldn't be triggered as long as it is empty.

Reply:
For rawkv and normal restoring logic, the way to process the range end key is different. I think it's better not to mix them together like how rawkv backup did...

Reply:
@MyonKeminta could you use `std::ops::Bound::Included`/`Excluded` to make it clear, like the rest of the code?

Reply:
Actually I think separating rawkv and transactional logic makes it more clear. PTAL again, thanks

Reply:
The code duplication is too much IMO...